Slider/Assets/Scripts/UI/Settings/BoolSettingUIElement.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BoolSettingUIElement : SettingUIElement<bool>
+{
+    
+}
```

Comment:
Unity serialization sucks x1

Reply:
this should probably be mentioned in the class